### ISSUE TITLE
Nice to meet you here

### ISSUE THREAD

**Issue by banjaminlee** [jnavarro30/playwright-portal#1]
Hi @jnavarro30 
Nice to meet you.
I am going to collaborate with you.
Can we have a talk ?
I am waiting here.

**Comment by jnavarro30** [comment]
Hello, it's just a testing project I'm slowly working on for a specific
website. What's your interest in it? What's your background?

**Comment by banjaminlee** [comment]
So are you working for a company right now?
I am looking for someone to collaborate with me.
I mean freelancing.

**Comment by jnavarro30** [comment]
Yes, I'm currently working for a company which leaves me little time to
work on other projects. I don't think I can be of much help.
